lms/djangoapps/instructor/views/api_urls.py

```diff
@@ -2,11 +2,17 @@
 Instructor API endpoint urls.
 """
 
-
 from django.conf.urls import url
 
 from lms.djangoapps.instructor.views import api, gradebook_api
 
+v0_api_urls = [
+    url(r'^tasks$', api.InstructorTasks.as_view(), name='list_instructor_tasks', ),
+    url(r'^reports$', api.ReportDownloads.as_view(), name='list_report_downloads', ),
+    url(r'^generate_report/problem_responses$', api.ProblemResponseReportInitiate.as_view(),
+        name='generate_problem_responses', ),
+]
+
```

Comment:
They are `v0` URLs only because they are just porting existing code. They've been around a long time and have stayed stable to might be good candidates for `v1`.

However, when the UI that uses these is ported to an MFE, these APIs can be reevaluated, and a decision can be made about whether these are good enough, or if we need new `v1` APIs the are better driven by MFEs.

Reply:
^ This comment may be worth including inline in the code itself too?

Reply:
Actually, while writing that comment I convinced myself to just call these `v1`. With `v0` the implication is that the API is still in development and may change in backwards-incompatible ways without bumping up the version number. That doesn't seem to be the case here. These APIs have been in use for a while without major changes.